jfrog-cli/artifactory/utils/docker/docker.go

```diff
@@ -146,33 +149,34 @@ func (getImageId *getParentId) GetEnv() map[string]string {
 func (getImageId *getParentId) GetStdWriter() io.WriteCloser {
 	return nil
 }
+
 func (getImageId *getParentId) GetErrWriter() io.WriteCloser {
 	return nil
 }
 
 // Get docker registry from tag
-func ResolveRegistryFromTag(imageTag string) string {
+func ResolveRegistryFromTag(imageTag string) (string, error) {
 	indexOfFirstSlash := strings.Index(imageTag, "/")
-	indexOfSecondSlash := strings.Index(imageTag[indexOfFirstSlash + 1:], "/")
-
-	// Should use docker hub
 	if indexOfFirstSlash < 0 {
-		return ""
+		err := errorutils.CheckError(errors.New("Invalid image tag received for pushing to Artifactory."))
```

Comment:
Let's append to this error message - *Tag does not include a slash.*

Reply:
Added